client/lib/get-selected-chart/index.js

```diff
@@ -0,0 +1,22 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { find } from 'lodash';
+
+/**
+ * Takes a chart name returns the configuration for that chart from and array of charts
+ *
+ * @param {string} chartName - the name of the chart to get configuration for
+ * @param {array} charts - list of charts for a particular report
+ * @returns {object} - chart configuration object
```

Comment:
May want to mention that if the chart name isn't found that it returns the first in the array.

Reply:
Added to the function description